starters/next-drupal-starter/__tests__/__snapshots__/umami/pages.test.jsx.snap

```diff
@@ -260,6 +260,8 @@ exports[`umami <PageTemplate /> > should render with umami profile page 1`] = `
             <p>
               Umami is a fictional food magazine that has been created to demonstrate how you might build a Drupal site using functionality provided 'out of the box'.
             </p>
+            
```

Comment:
Strange.

Reply:
I also thought this was pretty strange but digging into it more it makes sense. In our example data, there are some instances where `body.value` seems to have more newline characters than `body.processed`.

Example:
 https://github.com/pantheon-systems/decoupled-kit-js/blob/cec912d94683e1ff56fdec7e57d8ff5ea49367ad/starters/next-drupal-starter/__tests__/data/topLevelArticleData.json#L6

https://github.com/pantheon-systems/decoupled-kit-js/blob/cec912d94683e1ff56fdec7e57d8ff5ea49367ad/starters/next-drupal-starter/__tests__/data/topLevelArticleData.json#L8

Reply:
Hmm ok good to know thanks for digging that out!